# Bearing values

Header header

bool robotDetected

float32 absolute
float32 relative
